### ISSUE TITLE
How to cite your work

### ISSUE THREAD

**Issue by Shubodh** [unum-cloud/uform#43]
I can't find any research paper corresponding to this work. How can I cite your work in my research paper? I need it in the form of bibtex, for example like below:
```
@misc{shukor2022efficient,
      title={Efficient Vision-Language Pretraining with Visual Concepts and Hierarchical Alignment}, 
      author={Mustafa Shukor and Guillaume Couairon and Matthieu Cord},
      year={2022},
      eprint={2208.13628},
      archivePrefix={arXiv},
      primaryClass={cs.CV}
}
```

**Comment by ashvardanian** [comment]
There is a button for that on the main page of the repo. Would that work, @Shubodh?

![IMG_1063](https://github.com/unum-cloud/uform/assets/1983160/34e9cbce-c09e-4f16-b9d2-aeb49fb2fd60)

**Comment by kimihailv** [comment]
@Shubodh Hello! Thank you for your interest. 

1. Unfortunately, we doesn't have a paper. But we can reveal more details:

UForm is a two tower model with an image and text encoder (ViT for the image encoder, BERT for the text encoder, configs of models can be found in HF repositories). The text encoder consists of two parts – the unimodal part and the multimodal part. The unimodal part encodes a text using standard transformer layers with self-attention. The multimodal part additionally incorporates information from the image by cross-attention mechanism. The model was trained by minimizing three objectives: usual contrastive loss (unimodal part + image encoder), MLM (multimodal part + image encoder) and image text matching loss (multimodal part + image encoder). More details about objectives can be found in the paper "Align before Fuse: Vision and Language Representation Learning with Momentum Distillation". Training data: the part of CC + MSCOCO train split + VG + SBU. The multilingual model was trained on the balanced dataset with NLLB translations, special losses for cross-lingual properties were used.

2. We are not going to publish training coda. However our models are usual torch models, their weights are publicly available on HF. So you can re-train/fine-tune it like usual torch models.
